docs/pages/enterprise/workflow/resource-requests.mdx

```diff
@@ -108,7 +112,7 @@ Name                                 Hostname    Labels       Resource ID
 ------------------------------------ ----------- ------------ ------------------------------------------------------
 b1168402-9340-421a-a344-af66a6675738 iot         test=test    /teleport.example.com/node/b1168402-9340-421a-a344-af66a6675738
 bbb56211-7b54-4f9e-bee9-b68ea156be5f node        test=test    /teleport.example.com/node/bbb56211-7b54-4f9e-bee9-b68ea156be5f
- 
+
```

Comment:
I kind of doubt the `code` marker is responsible for dropping the empty lines, and the commands are meant to be copy-pasteable without the output, which works at the moment. We could just let it drop the lines, but the output ends up looking pretty cluttered imo

Reply:
@ptgott any idea what's going on here?

Reply:
I filed a bug https://github.com/gravitational/docs/issues/103

I think moving all the output into separate text sections would kind of break it up too much? I like seeing the command and output as it would appear in the terminal, and the `code` segment makes it easy to copy only the command without the output. I don't think the workaround of leaving spaces in there is that bad.

@zmb3 let me know if you'd like me to put the spaces back in for now and get this one merged

Reply:
Sure, sounds good.

Reply:
reverted the whitespace changes